go/app/main.go

```diff
@@ -25,17 +47,172 @@ func root(c echo.Context) error {
 	return c.JSON(http.StatusOK, res)
 }
 
+// dbを開く
+func opneSql(c echo.Context) (*sql.DB,  error){
+	read_file, err := sql.Open("sqlite3",db_file)
+	if err != nil {
+		c.Logger().Fatalf("Cannot open the db")
+	}
+	return read_file, err
+}
+
+// sql.RowsをItemListに変換
+func changeItemList(row_data *sql.Rows, c echo.Context) (ItemList) {
+	var items ItemList
+	for row_data.Next(){
+		var it Item
+		err := row_data.Scan(&it.Id, &it.Name, &it.Category, &it.ImageName)
+		if err != nil {
+			c.Logger().Fatalf("Cannot read the db data")
+		}
+		items.Items = append(items.Items, it)
+	}
+	return items
+}
+
+// imageのハッシュ生成
+func imageHash(img_file *multipart.FileHeader) (string, error) {
+	// 画像ファイルを開く
+	img, err := img_file.Open()
+	if err != nil {
+		fmt.Printf("Cannot open the image\n")
+	}
+	defer img.Close()
+
+	// hash値計算
+	hash := sha256.New()
+	if _, err := io.Copy(hash, img); err != nil {
+		fmt.Printf("Hash error\n")
```

Comment:
Errorはprintではなくloggerに書いてあげましょう。
また、errでが出たときこのさきに進めないコードであればreturnしたいです
```suggestion
c.Logger().Error("Hash error\n")
return nil, err
```